corehq/motech/repeaters/models.py

```diff
@@ -163,16 +164,27 @@ def register(self, payload, next_check=None):
         if not self.allowed_to_forward(payload):
             return
 
+        def forward_now(record):
+            from corehq.motech.repeaters.tasks import process_repeat_record
+
+            if not record.redis_lock.acquire(blocking=False):
+                return
+            process_repeat_record.delay(repeat_record)
+
         now = datetime.utcnow()
+        if next_check is None:
+            next_check = now
         repeat_record = RepeatRecord(
             repeater_id=self.get_id,
             repeater_type=self.doc_type,
             domain=self.domain,
             registered_on=now,
-            next_check=next_check or now,
+            next_check=next_check,
             payload_id=payload.get_id
         )
         repeat_record.save()
+        if next_check is now:
```

Comment:
Nit: I was briefly confused by this, but it seems that `is` is used instead of `==` to emphasize that this is checking whether `next_check` was just set to `now` in the if statement above, right? It might be a little more straightforward to keep using `next_check=next_check or now` above and check for `if next_check is None` on this line instead.

Reply:
Lol, in fact I considered leaving the code as it was, but chose `if next_check is now` because I imagined it was more readable. I'll go with what you're suggesting.